### ISSUE TITLE
Create own templates (breadcrumbs controller property)

### ISSUE THREAD

**Issue by luke-** [humhub/humhub-modules-yiigii#2]
When a new module is created using the Gii module generator, the error "Property "DefaultController.breadcrumbs" is not defined." is generated at line 4 of the modules index.php file.

Solution is to define breadcrumbs in the Controller class.

public $breadcrumbs = []

**Comment by baldwindc** [comment]
@luke- I agree with you. Best to have module specific code separate from HumHub code.
